e2e/LongHaul/service/DeviceOperations.cs

```diff
@@ -26,12 +25,10 @@ internal class DeviceOperations
 
         private static readonly TimeSpan s_retryInterval = TimeSpan.FromSeconds(1);
 
-        public DeviceOperations(IotHubServiceClient serviceClient, string deviceId, Logger logger)
+        public DeviceOperations(IotHubServiceClient serviceClient, string deviceId)
```

Comment:
Is there an advantage to passing the logger to each method instead of initializing it on the constructor itself?

Reply:
There will be custom dimension "OperationName" added in for each operation method, and will override its value if we use the same logger object.

Reply:
Maybe add this as a comment for context so that it is helpful when we look at the code later?

Reply:
We wouldn't need a comment if we cloned here and added the new property then passed it.